Fix the More Block input width by style #8750

miminari opened the pull request:
## Description
This PR fixes #8750.

## Screenshots <!-- if applicable -->
In Japanese
<img width="784" alt="スクリーンショット 2019-12-06 13 16 11" src="https://user-images.githubusercontent.com/487207/70295541-c0c43a80-182a-11ea-9744-d5e8a4737d0a.png">

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [x] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->

miminari committed:
一次訳　途中

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
一次訳完了

miminari committed:
Merge pull request #1 from miminari/translate/jp/design-principles

Translate/jp/design principles

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Update design-principles.md

miminari committed:
Fix the More Block input width by style #8750

youknowriad commented on packages/block-library/src/more/edit.js:
```diff
@@ -74,6 +75,7 @@ export default class MoreEdit extends Component {
 						size={ inputLength }
 						onChange={ this.onChangeInput }
 						onKeyDown={ this.onKeyDown }
+						style={ currentWidth }
```
Thanks for the PR. It seems that this doesn't work properly in Latin languages as the input is way too big. I'm not certain what's the correct way to fix this though.

miminari replied:
Thanks for the review!
I thought it would be better to see it than we couldn't see it. Just because we can't see the text, the function of this block looks incomplete in Japanese.

youknowriad replied:
I wonder if there's maybe a way we can design this block differently to avoid this issue. cc @jasmussen @mapk

jasmussen replied:
We can absolutely tweak the design to be more inclusive here, I can probably knock something together. 

Do you have a screenshot of what you're seeing, with the block being way too big? Cooking dinner as I write this so did not compile and check the branch.

jasmussen replied:
GIF:

![read more](https://user-images.githubusercontent.com/1204802/70516003-83b3bd00-1b36-11ea-81e8-8572ff5a18f2.gif)

Honestly I don't think it's the end of the world. Pending a better design of this, I think this would be a fine interim fix. 👍 👍

youknowriad replied:
That looks good yeah, I think while testing the "increase as you type" seemed more severe.

youknowriad commented on packages/block-library/src/more/edit.js:
```diff
@@ -74,6 +75,7 @@ export default class MoreEdit extends Component {
 						size={ inputLength }
```
I guess the size is unnecessary here right?

miminari replied:
If you adopt the way adding the style, I agree.

youknowriad replied:
Let's remove that and ship the PR.

mapk commented:
It's not bad and it does fix a language issue. 👍 

I did notice the text can continue endlessly... has that always been allowed?

![more 2019-12-16 18_02_14](https://user-images.githubusercontent.com/617986/70958515-567f7700-202e-11ea-8664-dfe021bc67a9.gif)

miminari committed:
Merge remote-tracking branch 'upstream/master'

draganescu commented:
hi @miminari can you remove that line from [Riad's comment](https://github.com/WordPress/gutenberg/pull/18966#discussion_r360790906) and then we can move on with this?

@mapk I tested that on master and it looks like that's how it's always been.

miminari committed:
Merge branch 'master' of github.com:miminari/gutenberg into fix_8750

miminari committed:
Update more tag style fixes #8750

miminari commented:
@draganescu Thank you for reminding me.
I removed the size.
And I would like to fix the overflow pointing out by @mapk, so I added the style `max-width:100%` to the input.

miminari commented:
I’m sorry, but I'd like to do the PR again for removing some unnecessary commits...
